api.md

```diff
@@ -778,17 +778,37 @@ Return the same Variant object for chaining.
 ```
 
 
-#### respondWithFile(filePath)
-* ***filePath***: the path to the file to serve out (any route variables can be used in the file path as well)
+#### respondWithFile(options)
+* ***options***: options to override the default values. *(These options can also be globally provided in the smocks core options when starting the server, see ([Starting the server](#section/Concepts/Starting%2520the%2520server))*
+  * ***mockDir***: The base directory where the mocking data exists. 
+    * *default value set to* ```./mocked-data```
+  * ***fileName***: The file name smocks will search for in the ***mockDir*** directory structure and then set as the response.
+    * *default value set to* ```response.json```
+  * ***statusCode***: The status code of the response.
+    * *Default value set to* ```200```
+    
+
+This method enables you to have your json files in a route similar directory structure as follows. 
 
 Remember that using ```./``` will refer to the top level module directory (the directory where ```node_modules``` exists regardless of the location of the file that is referring to a file location with ```./```);
 
+Example 1
 ```javascript
     var smocks = require('smocks');
-    smocks.route({path: '/customer/{id}'}).respondWithFile('./customer-{id}.json')
+    smocks.route({path: '/customer/{id}'}).respondWithFile()
```

Comment:
If no direct filename is provided I think we should have default lookup paths to make this more flexible.  so, with a route of `/customer/{a}/{b}` (using multiple tokens to represent n number of tokens).  This would allow for a more dynamic experience even with static files without sacrificing the simplicity of the current PR.

docs should be clear that you are referring to the route path rather than the route id (include an id which is required anyway in your `smocks.route options`.

We should look for files in the following order (notice we're removing the `{` and `}` from the route parts
1) /customer/a/b/`variant id`_`http method`-`path "a" value`-`path "b" value`.json
2) /customer/a/b/`variant id`_`http method`.json
3) /customer/a/b/`variant id`.json

So, for example, with a request of `/customer/123/456` (assuming the `GET` http method) and variant of `foo`
1) `/customer/a/b/foo_GET_123-456.json`
2) `/customer/a/b/foo_GET.json`
3)`/customer/a/b/foo.json`